lib/src/Uncertainty/Bayesian/Gibbs.cxx

```diff
@@ -134,6 +188,22 @@ Gibbs::MetropolisHastingsCollection Gibbs::getMetropolisHastingsCollection() con
   return samplers_;
 }
 
+/* Is order random accessors */
```

Comment:
Something like `setSamplingMethod` if you go for more than 2 sampling methods.

Reply:
I went for `setUpdatingMethod`. This is because articles often mention "deterministic" or "random" updating.